azkaban-common/src/main/java/azkaban/utils/Emailer.java

```diff
@@ -93,7 +93,7 @@ private void sendSlaAlertEmail(SlaOption slaOption, String slaMessage) {
           super.createEmailMessage(subject, "text/html", emailList);
 
       message.setBody(body);
-
+      message.setTLS(tls);
```

Comment:
I think this fix should be good enough – keep it simple. Let's see if repo owners have any comments.